01Matrix.js

```diff
@@ -0,0 +1,143 @@
+/*
+So we're going to represent our input as a grid where we're only concerned about the 1's, since we're trying 
+to find the distance for each 1 that is the closest to the nearest 0. 
```

Comment:
"since we're trying to find the distance for each 1 that is the closest to the nearest 0." This is a very confusing sentence, I have no idea what it means.

Reply:
I re-wrote it. Hopefully it makes sense:

Since we're trying to find the closest path to a 0 from each cell containing a 1, we're going to represent our input as a grid.

Reply:
Looks good!